test/functional/rpc_fundrawtransaction.py

```diff
@@ -777,10 +777,10 @@ def test_option_feerate(self):
         result4 = node.fundrawtransaction(rawtx, {"feeRate": str(10 * self.min_relay_tx_fee)})
 
         result_fee_rate = result['fee'] * 1000 / count_bytes(result['hex'])
-        assert_fee_amount(result1['fee'], count_bytes(result1['hex']), 2 * result_fee_rate)
-        assert_fee_amount(result2['fee'], count_bytes(result2['hex']), 2 * result_fee_rate)
-        assert_fee_amount(result3['fee'], count_bytes(result3['hex']), 10 * result_fee_rate)
-        assert_fee_amount(result4['fee'], count_bytes(result4['hex']), 10 * result_fee_rate)
+        assert_feerate_amount(result1['fee'], count_bytes(result1['hex']), 2 * result_fee_rate)
+        assert_feerate_amount(result2['fee'], count_bytes(result2['hex']), 2 * result_fee_rate)
+        assert_feerate_amount(result3['fee'], count_bytes(result3['hex']), 10 * result_fee_rate)
+        assert_feerate_amount(result4['fee'], count_bytes(result4['hex']), 10 * result_fee_rate)
```

Comment:
the first arg is `fee` (absolute fee), not a feerate

Reply:
oops. thanks, fixed, change is much smaller now